app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,23 @@
+class SessionsController < ApplicationController
+  def index
+  end
+
+  def create
+    user = User.from_omniauth(auth_info)
+    session[:user_id] = user.id
+    redirect_to '/dashboard'
+  end
+  
+  def destroy
+    session.delete(:user_id)
+    # session[:user_id] = nil
```

Comment:
We should remove this comment if we don't need it

Reply:
Got it!